src/scripts.js

```diff
@@ -149,6 +153,26 @@ function viewRecipe(event) {
   displaySelectedRecipeImg();
 }
 
+function viewRecipeFromIcon(event){
```

Comment:
Nice job getting this function working! I'd love to try and see if we can DRY it up from the function above it, but for now it is good and most important part is that it's working!